tests/lib/journalctl.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/sh
+
+JOURNALCTL_CURSOR_FILE="${SPREAD_PATH}"/journalctl_cursor
+
+get_last_journalctl_cursor(){
+    # It is not being used journalctl json output because jq is not installed on
+    # core systems
+    if [ -z "$(which jq)" ]; then
```

Comment:
I would recommend that we have one code path for all cases. The jq-free version looks fine so let's just use it all the time.